pkg/kvs/manifest/kvs.go

```diff
@@ -0,0 +1,86 @@
+package manifest
```

Comment:
put one conttainer impl under mock if needed 
work out one with manifest   
should  match the act container interface

Reply:
removed